Changelog.md

```diff
@@ -10,7 +10,10 @@ Compiler Features:
 
 
 Bugfixes:
-
```

Comment:
Not sure if it should have its own section like:
```
Antlr Grammar Changes:
 * Lexer: Fix of a discrepancy which allowed numbers followed by an identifier with no whitespace
 * Lexer: Fix divergent behaviour which allowed octal numbers.
 * Parser: More strict for free functions and will no longer accept some specifiers which are exclusive of contract functions.
```

Reply:
Let's follow the way it was done in older entries.